Gemfile

```diff
@@ -66,7 +66,7 @@ group :release do
   gem 'github_changelog_generator',  :require => false, :git => 'https://github.com/github-changelog-generator/github-changelog-generator' if RUBY_VERSION >= '2.2.2'
   gem 'puppet-blacksmith',           :require => false
   gem 'voxpupuli-release',           :require => false, :git => 'https://github.com/voxpupuli/voxpupuli-release-gem'
-  gem 'puppet-strings', '>= 1.0',    :require => false
+  gem 'puppet-strings', '>= 2.0',    :require => false
```

Comment:
Could you open a PR for this change [here](https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml#L93) instead?